custom-recipes/text-summarization-compute/recipe.py

```diff
@@ -69,10 +79,15 @@ def summarize(text):
             text = text.lower()
             all_capital = True
         
-        parser = PlaintextParser.from_string(text.decode(
-            'ascii', errors='ignore'), Tokenizer(LANGUAGE))
-        stemmer = Stemmer(LANGUAGE)
-
+        if PYTHON_VERSION == "PYTHON27":
+            parser = PlaintextParser.from_string(text.decode(
+                'ascii', errors='ignore'), Tokenizer(LANGUAGE))
+            stemmer = Stemmer(LANGUAGE)
+        else:
+            parser = PlaintextParser.from_string(text.encode().decode(
+                'ascii', errors='ignore'), Tokenizer(LANGUAGE))
+            stemmer = Stemmer(LANGUAGE)
+            
```

Comment:
Nice solution. I think we can either do it like you suggested, use the six package, or:
```python
import sys

PY2 = sys.version_info[0] == 2
PY3 = sys.version_info[0] == 3
```
(https://stackoverflow.com/questions/29213894/pythonic-way-to-ensure-unicode-in-python-2-and-3)

Maybe using sys is a bit less code, thoughts @JaneBellaiche @amandamilberg?



Also we can move the `stemmer` outside of the if / else I think to avoid duplicating code

Reply:
I agree, good idea Niklas 👍